### ISSUE TITLE
Configure automated code review

### ISSUE THREAD

**Issue by italopessoa** [ifactory-solutions/inside-client#25]
Hi, guys!
What do you think about adding an automated code review to the project? I'm currently using [Codacy](www.codacy.com) on personal projects. It helps me a lot. It uses ESlint rules and some other that they detect automatically depending on what languages your project is using. Security issues, Erro prone, complexity, code coverage, project quality (we already have it on tra1vis).
Integration with Github and Slack as well.
Free for open source projects

For more details: https://www.codacy.com/features

**Comment by jvcjunior** [comment]
Hey @italopessoa, is that possible to add the same eslint rules as we have in our project?

**Comment by italopessoa** [comment]
Absolutely @jvcjunior, as you can see below, codacy will look for an existing eslint file before applying its own rules.
![image](https://user-images.githubusercontent.com/9323529/31015322-df6c9a7c-a4f5-11e7-87c2-cb16839e8ea7.png)

**Comment by jourdanrodrigues** [comment]
I'm really cool with the concept. I once used [Code Climate](https://codeclimate.com/), but Codacy seems to be more specific for this purpose.

About the integration with ESLint specifically, see [this post](https://blog.codacy.com/codacy-integrates-with-eslint-6e757b6c802d) in their blog.

**Comment by jvcjunior** [comment]
Awesome!!!

**Comment by jvcjunior** [comment]
@italopessoa go ahead if you want!

**Comment by italopessoa** [comment]
@jvcjunior it's necessary to create a codacy account, they have integrated login using Github. What do I do?

**Comment by paulonotz0r** [comment]
Awesome! 😍

**Comment by jourdanrodrigues** [comment]
@italopessoa I'm on it.

**Comment by jvcjunior** [comment]
I already created @jourdanrodrigues! It's online already. Go ahead and take a look at it using your github account on codacy

**Comment by jourdanrodrigues** [comment]
@jvcjunior Think there's something weird happening here.

I managed to create the app in Codacy on **my own user**;
Then, I created an organization and transferred the project to it;
Then you said you already did the same thing I did, at the same time.

@italopessoa Shouldn't this situation have generated a conflict or something telling that this repo is already set? 🤔 

I'm not familiar with Codacy, sorry if said something stupid.
